Source/HTTP/httpcallresponse.cpp

```diff
@@ -23,10 +22,26 @@ HCHttpCallResponseSetResponseString(
     _In_ PCSTR_T responseString
     )
 {
-    verify_http_singleton();
     call->responseString = responseString;
-    HC_TRACE_INFORMATION(HTTPCLIENT, "HCHttpCallResponseSetResponseString [ID %llu]: responseString=%s",
-        call->id, responseString);
+
+#if HC_TRACE_ENABLE
+
+#if HC_CHAR_T_IS_WIDE
```

Comment:
Consider adding an tstrlen function that is defined for the correct variant of CHAR_T to avoid this split code (look at the various printf variants in trace.cpp for what I mean)